f8a_worker/workers/githuber.py

```diff
@@ -114,12 +109,10 @@ def execute(self, arguments):
         else:
             # there might be more comma-separated tokens, randomly select one
             token = random.choice(token.split(',')).strip()
```

Comment:
Maybe a method for this in the configuration as well so it is done at one place consistently.